### PR TITLE
[WIP]add new files to ants brain extraction output spec and list outputs

### PR DESCRIPTION
**Pull request by rwblair**
For now exposing all files generated by ants brain extraction. Of particular interest are the n4 bias correction and segmentation mask.

### CODE REVIEW

File: nipype/interfaces/ants/segmentation.py [hunk 1]
```diff
@@ -701,7 +701,7 @@ def _list_outputs(self):
                                                        self.inputs.out_prefix +
                                                        'BrainExtractionBrain.' +
                                                        self.inputs.image_suffix)
-        if isdefined(self.inputs.keep_temporary_files) && self.inputs.keep_temporary_files != 0:
+        if isdefined(self.inputs.keep_temporary_files) and self.inputs.keep_temporary_files != 0:
```
Comment (oesteban) [inline nipype/interfaces/ants/segmentation.py]: hehehe I was to say that I preferred an "and" (the word) there.

Comment (rwblair) [inline nipype/interfaces/ants/segmentation.py]: thats what I get for having syntax highlighting turned off...

**Comment by chrisfilo** [comment]
LGTM - anything else missing here?

**Comment by satra** [comment]
@chrisfilo - we should start building out regression tests soon for interfaces - so we may ask people who augment or add interfaces to provide a regression test.

**Comment by chrisfilo** [comment]
By that you mean a test that runs the software a particular interface is
wrapping? Running those will take a lot of time...

**Comment by satra** [comment]
yes - essentially as part of repronim we want to start doing regression tests for the underlying software. these won't run by default, but we will be creating various containers to test the underlying tools across versions of the tools and there will be a dashboard reflecting these tests.

**Pull request merged**